source/module_cell/unitcell.cpp

```diff
@@ -58,7 +58,7 @@ UnitCell::~UnitCell()
 	delete[] lc;
 }
 
-
+#include "../src_parallel/parallel_common.h"
```

Comment:
Shall this line wrapped in `#ifdef __MPI`?

Reply:
No